src/components/TeamForm/TeamForm.js

```diff
@@ -59,6 +66,22 @@ class TeamForm extends Component {
             </Col>
           </Row>
         </FormGroup>
+        <FormGroup controlId="teamForm-vote-duration">
+          <ControlLabel for="voteDuration">Sort Duration</ControlLabel>
+          <p>Sort duration refers to the amount of time votes and decisions factor in to how restaurants are sorted.  For example, if you choose Burger King for today's lunch (yum!) and your sort duration is set to 7 days, Burger King will appear towards the bottom of your restaurant list for the next week.</p>
```

Comment:
Please use a `<HelpBlock />` for this text and place it underneath the following `<Row />`. And although it might get a little wordy, you might also consider providing another example relating to voting (versus making a decision) and how it moves un-decided, highly-voted restaurants to the top. (Maybe put each example on a different line using `<br />`s.

Reply:
And while I personally think the use of Burger King and "(yum!)" is cute, I'd prefer to stay away from preferential treatment - maybe change it to "Burger Monarch" or something similarly evocative (sadly, with no yum)?